frontend/template-partials/views/partials/gatag.html

```diff
@@ -59,3 +59,25 @@
     }
   </script>
 {{/ga4TagId}}
+
+{{#gtmTagId}}
+  <!-- Data Layer -->
```

Comment:
what do you mean by Data layer? was this comment from you?

Reply:
it's from the snippet code sent from the business. I think it's just a comment to explain what the code is for.